src/components/Navbars/DashboardNavbar.js

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+const isMvp = process.env.REACT_APP_MVP_SETTING;
```

Comment:
I've found the `.env` doesn't do booleans. It's silly looking, but you have to do this to get it to work.

`const isMvp = process.env.REACT_APP_MVP_SETTING === 'false';`

Reply:
Dun did.